alb-maintenance-window.cfndsl.rb

```diff
@@ -42,7 +44,7 @@
     }
   ]
 
-  name = "MaintenanceWindowRule#{priority}"
+  name = "MaintenanceWindowRule#{SecureRandom.hex}"
```

Comment:
this will cause the resource to be recreated each time but it will fail due to it having the same priority.